notebooks/Liquidators_data.ipynb

```diff
@@ -0,0 +1,1325 @@
+    "def hex_to_int(row):\n",
+    "    token = row['debt_token']\n",
+    "    decimal = token_info[token][1]\n",
+    "    face_amount = int(row['debt_face_amount'], 0) / (10**decimal)\n",
+    "\n",
+    "    if token_info[token][0] == \"ETH\":\n",
+    "#         print(row)\n",
+    "        token_price = row['Adj_Close_ETH']\n",
+    "    elif token_info[token][0] == \"USDC\":\n",
+    "        token_price = row['Adj_Close_USDC']\n",
+    "    elif token_info[token][0] == \"USDT\":\n",
+    "        token_price = row['Adj_Close_USDT']\n",
+    "    elif token_info[token][0] == \"BTC\":\n",
+    "        token_price = row['Adj_Close_BTC']\n",
+    "    else:\n",
+    "        token_price = None  # handle unknown token\n",
+    "    \n",
+    "    return face_amount * token_price if token_price else None\n",
+    "\n",
+    "# Apply the function to the dataframe and create a new column\n",
+    "vertical_stack_final['debt_usd_value'] = vertical_stack_final.apply(hex_to_int, axis=1)\n",
+    "\n",
+    "# print(vertical_stack_final[vertical_stack_final['Protocol'] == 'hashstack_v1'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d41fa4ff",
+   "metadata": {},
+   "source": [
+    "# 3 Average liquidated debt"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 337,
+   "id": "da3d4ed8",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Average Liquidated Debt Amount (All): 271.4080187782815\n",
+      "\n",
+      "Average Liquidated Debt Amount per Protocol:\n",
+      "Protocol\n",
+      "hashstack_v0     75.856113\n",
+      "hashstack_v1    649.490061\n",
+      "zklend            4.329041\n",
+      "Name: debt_usd_value, dtype: float64\n",
+      "\n",
+      "Average Liquidated Debt Amount per Token:\n",
+      "debt_token\n",
+      "0x3fe2b97c1fd336e750087d68b9b867997fd64a2661ff3ca5a7c771641e8e7ac    2.739593e-09\n",
+      "0x49d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7    1.818003e+02\n",
+      "0x53c91253bc9682c04929ca02ed00b3e423f6710d2ee7e0d5ebb06f3ecf368a8    3.420048e+02\n",
+      "0x68f5c6a61780768455de69077e07e89787839bf8166decfbf92b645209c0fb8    1.026264e+03\n",
+      "Name: debt_usd_value, dtype: float64\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Average liquidated debt amount across all liquidations\n",
+    "average_debt_all = vertical_stack_final['debt_usd_value'].mean()\n",
+    "print(\"Average Liquidated Debt Amount (All):\", average_debt_all)\n",
+    "\n",
+    "# Average liquidated debt amount per protocol\n",
+    "average_debt_per_protocol = vertical_stack_final.groupby('Protocol')['debt_usd_value'].mean()\n",
+    "print(\"\\nAverage Liquidated Debt Amount per Protocol:\")\n",
+    "print(average_debt_per_protocol)\n",
+    "\n",
+    "# Average liquidated debt amount per token\n",
+    "average_debt_per_token = vertical_stack_final.groupby('debt_token')['debt_usd_value'].mean()\n",
```

Comment:
Can you pls print the human-readable tickers instead of addresses? (holds for all tokens in all charts/tables) 🙏🏼